Fix IPC and boot timeouts

kv2019i opened the pull request:
Fix duplicated definition of IPC timeout value in ipc.c&core.c plus adjust the defaults to much higher values. This allows to use the driver out-of-the-box on QEMU. I don't see any functional needs to have very short timeout values.

plbossart commented on sound/soc/sof/core.c:
```diff
@@ -23,8 +23,8 @@ MODULE_PARM_DESC(sof_debug, "SOF core debug options (0x0 all off)");
-#define TIMEOUT_DEFAULT_IPC_MS  5
-#define TIMEOUT_DEFAULT_BOOT_MS 100
+#define TIMEOUT_DEFAULT_IPC_MS  3000
+#define TIMEOUT_DEFAULT_BOOT_MS 5000
```
Yes, but the message really is "please define platform-specific constraints otherwise you will use defaults"

kv2019i replied:
@plbossart Ack, point taken, fixed in the new patch,

kv2019i commented:
Patchset updated to address Pierre's comments. 

I reduced the maximum timeout to 2sec, to align it with our other "long delays" (like the 2 second runtime suspend default). Other reason is to allow the timeouts to kick in typical system suspend/resume stress tests, where the the system uptime is typically 5...30sec or more. If our IPC timeouts are longer and we get IPC failures in the stress tests, the timeout information might be lost if we had very long default timeout (tens of seconds). Third, duration of IPC execution migh tbe very long in QEMU, so it will not be sufficient to just to icnrease these defaults. So as a summary, defaults of 2sec and 500ms seem reasonable.

Please review.

ranj063 commented:
@kv2019i missing "SOF" in commit prefix and also 2 "no"'s.

plbossart commented:
> @kv2019i missing "SOF" in commit prefix and also 2 "no"'s.

what @ranj063 said

kv2019i commented:
@ranj063 > @kv2019i missing "SOF" in commit prefix and also 2 "no"'s.

Oops, dunno how that slipped in. Fixed in the new patches. Thanks for the review!^
UPDATE: the 2nd patch was also missing in my previous push, now added back.

ranj063 commented:
@kv2019i needs to be rebased

kv2019i commented:
> @kv2019i needs to be rebased

Ack @ranj063 , done
